esmvalcore/_config.py

```diff
@@ -22,6 +22,9 @@ def find_diagnostics():
         import esmvaltool
     except ImportError:
         return Path.cwd()
+    # catch a lingering failed install
```

Comment:
```suggestion
    # avoid a crash when there is a directory called 'esmvaltool' that is not a Python package
```
Good find, though I think that the actual explanation is that there is an empty directory called `esmvaltool` either in your current working directory or somewhere where Python is looking for modules. That would get interpreted as a [namespace package](https://packaging.python.org/guides/packaging-namespace-packages/#native-namespace-packages) and confuse the tool.

Reply:
yup good call Bouwe man, just plugged in the change :+1: